app/templating/AssetHelper.scala

```diff
@@ -31,14 +31,27 @@ trait AssetHelper extends HasEnv { self: I18nHelper with SecurityHelper =>
     cssTagWithDirAndTheme(name, isRTL(using ctx.lang), ctx.currentBg)
 
   def cssTagWithDirAndTheme(name: String, isRTL: Boolean, theme: String): Frag =
-    cssAt(s"css/$name.${if (isRTL) "rtl" else "ltr"}.$theme.${if (minifiedAssets) "min" else "dev"}.css")
+    if (theme == "system")
+      frag(
+        cssTagWithDirAndSimpleTheme(name, isRTL, "light")(media := "(prefers-color-scheme: light)"),
+        cssTagWithDirAndSimpleTheme(name, isRTL, "dark")(media := "(prefers-color-scheme: dark)")
+      )
+    else cssTagWithDirAndSimpleTheme(name, isRTL, theme)
+
+  private def cssTagWithDirAndSimpleTheme(name: String, isRTL: Boolean, theme: String): Tag =
+     cssAt(s"css/$name.${if (isRTL) "rtl" else "ltr"}.$theme.${if (minifiedAssets) "min" else "dev"}.css")
 
   def cssTagNoTheme(name: String): Frag =
     cssAt(s"css/$name.${if (minifiedAssets) "min" else "dev"}.css")
 
-  private def cssAt(path: String): Frag =
+  private def cssAt(path: String): Tag =
     link(href := assetUrl(path), rel := "stylesheet")
 
+  val systemThemePolyfillJs = """
+if (window.matchMedia('(prefers-color-scheme: dark)').media === 'not all')
+    document.querySelectorAll('[media="(prefers-color-scheme: dark)"]').forEach(e=>e.media='')
+"""
+
```

Comment:
ok, so turns out, iOS Safari 12 doesn't support `prefers-color-scheme` ...

Annoyingly, there doesn't seem to be any way to tell get it to load one of the stylesheets anyway. Media attributes involving unknown stuff just never match, which also means there's not even a way to have a third stylesheet that the others would ignore.

This JS poly-fill which just falls it back to dark mode seems to be the best solution I was able to find and as far as I'm able to tell, it doesn't really have any noticeable impact?